### PR TITLE
merge stg to prod

### PR DESCRIPTION
**Pull request by huybuidoanquang**

### CODE REVIEW

File: aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/ReportEducationHaveCVPassInterViewDto.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TalentV2.DomainServices.Reports.Dtos
+{
+    public class ReportEducationHaveCVPassInterViewDto : ReportEducationHaveCVPassTestDto
+    {
```
Comment (duynxncc) [inline aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/ReportEducationHaveCVPassInterViewDto.cs]: why does it exist?

File: aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/OverviewHiringDto.cs [hunk 2]
```diff
@@ -38,4 +39,47 @@ public class StatusStatistic
...
+
+    public class Templates
+    {
+        public string Key { get; set; }
+        public float Percent { get; set; }
+    }
+    public class BranchDtoExport
+    {
+        public long? id { get; set; }
+        public string displayName { get; set; }
```
Comment (duynxncc) [inline aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/OverviewHiringDto.cs]: pascal case

File: angular/src/shared/shared.module.ts [hunk 3]
```diff
@@ -105,6 +105,11 @@ import { SectionBreadcrumbComponent } from './components/section-breadcrumb/sect
 import { PositionDropdownComponent } from './components/position-dropdown/position-dropdown.component';
 import { PageErrorComponent } from './pages/page-error/page-error.component';
 import {SafeHtmlPipe} from '@shared/pipes/safe-html.pipe';
+
+import {ExportCandidateComponent} from './components/export-candidate/export-candidate.component'
+import {TabsModule} from "ngx-bootstrap/tabs";
+import {ExportDialogComponent} from './components/export-dialog/export-dialog.component'
```
Comment (duynxncc) [inline angular/src/shared/shared.module.ts]: format consitency, space after and before {}

File: angular/src/app/modules/admin/mail/edit-mail-dialog/edit-mail-dialog.component.html
Comment (khanh-lesy) [inline angular/src/app/modules/admin/mail/edit-mail-dialog/edit-mail-dialog.component.html:1]: ddd

**Review by duynxncc — CHANGES_REQUESTED** [review]

File: angular/src/shared/shared.module.ts (hunk 3 above)
Comment (khanh-lesy) [reply angular/src/shared/shared.module.ts]: yes, I will pay more attention

File: aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/OverviewHiringDto.cs (hunk 2 above)
Comment (khanh-lesy) [reply aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/OverviewHiringDto.cs:72]: yes, I will pay more attention

File: aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/ReportEducationHaveCVPassInterViewDto.cs (hunk 1 above)
Comment (khanh-lesy) [reply aspnet-core/src/TalentV2.Core/DomainServices/Reports/Dtos/ReportEducationHaveCVPassInterViewDto.cs]: Do I need to repair it?

**Review by duynxncc — APPROVED** [review]

**Pull request merged**